test/verify/check-connection

```diff
@@ -255,13 +255,13 @@ class TestConnection(MachineCase):
 
         # login handler: wrong password
         headers = m.execute("curl -s --head --header 'Authorization: Basic {}' http://172.27.0.15:9090/cockpit/login".format(
-            base64.b64encode(b"admin:hahawrong")))
+            base64.b64encode(b"admin:hahawrong").decode()))
```

Comment:
b64encode returns bytes?!  this is some kind of awful :(

(I checked, and indeed, it seems that there is no better way to handle this)